mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -71,6 +71,22 @@ private[recommendation] trait ALSModelParams extends Params with HasPredictionCo
 
   /** @group getParam */
   def getItemCol: String = $(itemCol)
+
+  /**
+   * Param for strategy for dealing with unknown users or items at prediction time.
+   * Supported values:
+   * - "nan": prediction value for unknown ids will be NaN.
+   * - "drop": rows in the input DataFrame containing unknown ids will be dropped.
+   * Default: "nan".
+   * @group expertParam
+   */
+  val unknownStrategy = new Param[String](this, "unknownStrategy",
```

Comment:
@MLnick I think it's good. It's an "expert" param anyway. Also, it's better in a way because it's hard to _assume_ what it means. For those that don't know it, they can check out the doc.